packages/jest-matchers/package.json

```diff
@@ -13,6 +13,7 @@
     "jest-get-type": "^20.0.3",
     "jest-matcher-utils": "^20.0.3",
     "jest-message-util": "^20.0.3",
-    "jest-regex-util": "^20.0.3"
+    "jest-regex-util": "^20.0.3",
+    "lodash.partition": "^4.6.0"
```

Comment:
I don't want this dependency. Can you inline it or find something that isn't lodash? Thanks.